apps/ledger-live-desktop/src/newArch/Collectibles/types/Media.ts

```diff
@@ -16,18 +16,20 @@ type BaseMediaProps = BaseProps & {
   previewUri?: string | undefined;
   originalUri?: string | undefined;
   mediaType: string | undefined;
-  setUseFallback: (a: boolean) => void;
+  setUseFallback?: (a: boolean) => void;
```

Comment:
not a good practice to use single char const :)

Reply:
Changed for `(useFallBack: boolean)`